src/odemis/driver/xt_client.py

```diff
@@ -612,6 +624,144 @@ def set_autostigmator(self, channel_name, state):
+        as multi-probe shape. Each stigmator has two control values; x and y.
+
+        Returns
+        -------
+        tuple, (x, y) control values of primary stigmator, unitless.
+        """
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.get_primary_stigmator()
+
+    def set_primary_stigmator(self, x, y):
+        """
+        Set the control values of the primary stigmator. Within the MBSEM system
+        there are two stigmators to correct for both beamlet astigmatism as well
+        as multi-probe shape. Each stigmator has two control values; x and y.
+
+        Parameters
+        -------
+        (x, y) control values of primary stigmator, unitless.
+        """
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.set_primary_stigmator(x, y)
+
+    def primary_stigmator_info(self):
+        """"Returns the unit and range of the primary stigmator."""
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.primary_stigmator_info()
+
+    def get_secondary_stigmator(self):
+        """
+        Get the control values of the secondary stigmator. Within the MBSEM system
+        there are two stigmators to correct for both beamlet astigmatism as well
+        as multi-probe shape. Each stigmator has two control values; x and y.
+
+        Returns
+        -------
+        tuple, (x, y) control values of primary stigmator, unitless.
+        """
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.get_secondary_stigmator()
+
+    def set_secondary_stigmator(self, x, y):
+        """
+        Get the control values of the secondary stigmator. Within the MBSEM system
+        there are two stigmators to correct for both beamlet astigmatism as well
+        as multi-probe shape. Each stigmator has two control values; x and y.
+
+        Parameters
+        -------
+        (x, y) control values of primary stigmator, unitless.
+        """
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.set_secondary_stigmator(x, y)
+
+    def secondary_stigmator_info(self):
+        """"Returns the unit and range of the secondary stigmator."""
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.secondary_stigmator_info()
+
+    def get_dc_coils(self):
+        """
+        Get the four values of the dc coils.
+
+        Returns
+        -------
+        list, len 4
+            These 4 items describe 4x2 transformation matrix for 1 um beam shift using DC coils.
+        """
+        with self._proxy_access:
+            self.server._pyroClaimOwnership()
+            return self.server.get_dc_coils()
+
+    def get_use_case(self):
```

Comment:
This name does not directly reveal, what actually is meant with use case. However, if we do implement a proper API for all the xt functionality, then we could just do it there, and leave here the name the same as the xt call. Otherwise, I would suggest something like "get_sem_mode".

Reply:
Lets leave the name as it is for now, and give it a proper name in the API. I will update the docstring to better explain the method.

Reply:
I think that the names should be just the same as the names used by the xtLib/xtToolkit. The xtadapter is not really try to hide the low-level API and provide a high-level view, but just to pass the messages as-is.
With all this said... I agree with Sabrina that here "usecase" is a really bad name. Such a bad name that it'd be fine to bend the rule, and pick a better name ;-)